tests/e2e/main_test.go

```diff
@@ -13,19 +13,32 @@ import (
 )
 
 const (
-	keplerImage = `quay.io/sustainable_computing_io/kepler:release-0.7.12`
+	keplerImage       = `quay.io/sustainable_computing_io/kepler:release-0.7.12`
+	keplerRebootImage = `quay.io/sustainable_computing_io/kepler-reboot:v0.0.4`
+	ciTestVMEnvKey    = `powermonitor.sustainable.computing.io/test-env-vm`
 )
 
 var (
-	Cluster         k8s.Cluster = k8s.Kubernetes
-	testKeplerImage string
+	Cluster               k8s.Cluster = k8s.Kubernetes
+	testKeplerImage       string
+	testKeplerRebootImage string
+	vmAnnotationKey       string
+	enableVMEnv           string
 )
 
 func TestMain(m *testing.M) {
 	openshift := flag.Bool("openshift", true, "Indicate if tests are run aginast an OpenShift cluster.")
 	flag.StringVar(&controller.KeplerDeploymentNS, "deployment-namespace", controller.KeplerDeploymentNS,
 		"Namespace where kepler and its components are deployed.")
 	flag.StringVar(&testKeplerImage, "kepler-image", keplerImage, "Kepler image to use when running Internal tests")
+	flag.StringVar(&testKeplerRebootImage, "kepler-reboot-image", keplerRebootImage, "Kepler image to use when running PowerMonitorInternal tests")
+	flag.StringVar(&vmAnnotationKey, "vm-annotation-key", ciTestVMEnvKey, "VM Annotation Key set to enable vm test environment")
+	enableFakeMeter := "false"
+	if os.Getenv("ENABLE_VM_TEST") == "true" {
+		enableFakeMeter = "true"
+	}
```

Comment:
lets not make use of env vars when we can use `args`. My objection against ENV vars is that it modifies behavior silently